web/src/containers/activity/CostAllocateFFPQuarterly.js

```diff
@@ -190,12 +199,14 @@ CostAllocateFFPQuarterly.propTypes = {
 const makeMapStateToProps = () => {
   const selectCostAllocateFFPBudget = makeSelectCostAllocateFFPBudget();
   const mapStateToProps = (state, props) =>
+    ariaQueuedMessage: getAriaQueuedAnnouncement(state),
```

Comment:
This approach seems like it'll work, but I can't test because this fails -- I can't figure out how to properly merge it with the object that `selectCostAllocateFFPBudget` returns.